setup.cfg

```diff
@@ -19,10 +19,13 @@ max-line-length = 88
 ignore = E203, E266, E501, W503
 select = B,C,E,F,W,T4,B9
 per-file-ignores =
-    modin/pandas/__init__.py:E402
+    modin/pandas/__init__.py:E402,F401
```

Comment:
Why is the `F401` needed? Unused import?

Reply:
flake8 says `partition_format` is unused while it is actually used in `_update_engine`.